README.rst

```diff
@@ -411,9 +408,6 @@ runpy scripts).
    * - interactive interpreter with deps
      - ✓
      -
-   * - re-use existing environment
-     - ✓
-     -
```

Comment:
On further consideration, this feature still exists and still distinguishes pip-run from pipx. Even though existing satisfied dependencies will be overlayed, the existing target environment will still be present, and unspecified dependencies will be available from that environment.